hackathon_site/dashboard/frontend/src/pages/TeamDetail/TeamDetail.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { screen } from "@testing-library/react";
+import { render } from "testing/utils";
+
+import { mockHardware, mockTeamMultiple } from "testing/mockData";
+
+import TeamDetail from "pages/TeamDetail/TeamDetail";
+import { Provider } from "react-redux";
+
+describe("User info table", () => {
+    test("renders user info table", () => {
```

Comment:
It's great you're breaking up the tests but you can combine all these unit tests into one because you're testing the same component. You can make all the assertions in a single forEach() loop instead of multiple